routing/localchans/manager.go

```diff
@@ -0,0 +1,100 @@
+package localchans
+
+import (
+	"sync"
+
+	"github.com/btcsuite/btcd/wire"
+	"github.com/lightningnetwork/lnd/channeldb"
+	"github.com/lightningnetwork/lnd/discovery"
+	"github.com/lightningnetwork/lnd/htlcswitch"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/routing"
+)
+
+// Manager manages the node's local channels. The only operation that is
+// currently implemented is updating forwarding policies.
+type Manager struct {
```

Comment:
Nice set up for future changes in this area which may have automated updates triggered due to fees on chain, off-chain fees, time locks, etc.

Reply:
Didn't think about that yet. But indeed, we are seeing node operators starting to use dynamic policies based on the state of a channel.